src/Core.zig

```diff
@@ -40,25 +40,61 @@ windows: mach.Objects(
         /// Window title string
         // TODO: document how to set this using a format string
         // TODO: allocation/free strategy
-        title: []const u8,
+        title: []const u8 = "Mach Window",
 
         /// Texture format of the framebuffer (read-only)
-        framebuffer_format: gpu.Texture.Format,
+        framebuffer_format: gpu.Texture.Format = .bgra8_unorm,
 
         /// Width of the framebuffer in texels (read-only)
-        framebuffer_width: u32,
+        framebuffer_width: u32 = 1920 / 2,
 
         /// Height of the framebuffer in texels (read-only)
-        framebuffer_height: u32,
+        framebuffer_height: u32 = 1080 / 2,
```

Comment:
Interestingly, we don't actually know the framebuffer dimensions at this point in time. I think having these defaults is good, but let's add a comment like:

```
/// Height of the framebuffer in texels (read-only)
/// Will be updated to reflect the actual framebuffer dimensions after window creation.
```

(and same for `framebuffer_width`)